### PR TITLE
UTF 8 encode response for python3 api responses

### PR DESCRIPTION
**Pull request by bbchungg**
Response weren't being formatted properly in python3.  Added an attempt to encode the responses in a backwards compatible way.

### CODE REVIEW

File: riskiq/cli/dns.py [hunk 1]
```diff
@@ -24,14 +25,14 @@ def get_data(client, cmd, rrtype=None, hostname=None, ip=None):
     data = None
     if cmd == 'name':
         if ip is not None:
-            data = client.get_dns_ptr_by_ip(ip, rrtype=rrtype)
+            data = client.get_dns_ptr_by_ip(ip, rrtype=rrtype,maxresults=500)
```
Comment (nitinbhatt) [inline riskiq/cli/dns.py]: Arguments should be separated by comma followed by a space.
=> rrtyte, maxResults

File: riskiq/cli/dns.py [hunk 2]
```diff
@@ -24,14 +25,14 @@ def get_data(client, cmd, rrtype=None, hostname=None, ip=None):
...
         elif hostname is not None:
             data = client.get_dns_data_by_name(hostname, rrtype=rrtype)
         else:
             raise ValueError('No IP or hostname')
     elif cmd == 'data':
         if ip is not None:
-            data = client.get_dns_data_by_ip(ip, rrtype=rrtype)
+            data = client.get_dns_data_by_ip(ip, rrtype=rrtype,maxresults=500)
```
Comment (nitinbhatt) [inline riskiq/cli/dns.py]: Arguments should be separated by comma followed by a space.

**Review by nitinbhatt — CHANGES_REQUESTED** [review]
FTP

**Review by nitinbhatt — APPROVED** [review]
LGTM

**Review by jlaycock-riq — APPROVED** [review]
Looks good, maybe need to be applied to other locations

**Pull request merged**